tests/application/cms/test_models.py

```diff
@@ -930,3 +931,13 @@ def test_to_dict(self):
             "settings_and_source_data": {},
             "table_object": {"table": "object"},
         }
+
+
+class TestTopicModel:
+    def test_topic_with_short_title(self):
+        topic = TopicFactory.build(title="Long title", short_title="Short title")
```

Comment:
👍

I think you might need to add `short_title` to the `TopicFactory` model, though?